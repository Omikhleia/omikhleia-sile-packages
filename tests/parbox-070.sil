\begin{document}
\neverindent
\nofolios
\script[src=packages/parbox]
\script[src=packages/lorem]
% Test case 070 = From the package documentation, regular vs. minimized

(7A) \parbox[width=37%fw, valign=middle, border=0.5pt]{First\par
37\%fw width\par
Not minimized}

\smallskip

(7B) \parbox[width=37%fw, valign=middle, border=0.5pt, minimize=true]{Second\par
37\%fw width\par
Minimized}

\end{document}