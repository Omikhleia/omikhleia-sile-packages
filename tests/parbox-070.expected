Set paper size 	595.275597	841.8897729
Begin page
Mx 	29.7638
My 	57.4783
Set font 	Gentium Plus;10;400;;normal;;LTR
T	11 w=3.1689	(()
Mx 	32.9327
T	26 36 w=10.6445	(7A)
Mx 	43.5773
T	12 w=3.1689	())
Draw line	49.4238	42.0945	198.2268	0.5000
Draw line	49.4238	72.8621	198.2268	0.5000
Draw line	49.4238	42.0945	0.5000	31.2676
Draw line	247.6506	42.0945	0.5000	31.2676
Mx 	49.4238
My 	48.7156
T	41 76 85 86 87 w=18.7549	(First)
My 	60.7156
T	22 26 w=9.3848	(37)
Mx 	58.8085
T	8 w=6.8994	(%)
Mx 	65.7080
T	73 90 w=10.1709	(fw)
Mx 	78.5374
T	90 76 71 87 75 w=23.9307	(width)
Mx 	49.4238
My 	72.7156
T	49 82 87 w=14.9609	(Not)
Mx 	67.0424
T	80 76 81 76 80 76 93 72 71 w=43.7744	(minimized)
Mx 	29.7638
My 	92.7195
T	11 w=3.1689	(()
Mx 	32.9327
T	26 37 w=10.1904	(7B)
Mx 	43.1232
T	12 w=3.1689	())
Draw line	48.9857	76.8621	53.0283	0.5000
Draw line	48.9857	108.5769	53.0283	0.5000
Draw line	48.9857	76.8621	0.5000	32.2148
Draw line	102.0140	76.8621	0.5000	32.2148
Mx 	48.9857
My 	84.4304
T	54 72 70 82 81 71 w=29.4971	(Second)
My 	96.4304
T	22 26 w=9.3848	(37)
Mx 	58.3705
T	8 w=6.8994	(%)
Mx 	65.2699
T	73 90 w=10.1709	(fw)
Mx 	78.0834
T	90 76 71 87 75 w=23.9307	(width)
Mx 	48.9857
My 	108.4304
T	48 76 81 76 80 76 93 72 71 w=43.8037	(Minimized)
End page
Finish
